src/main/perl/Reporter.pm

```diff
@@ -202,6 +202,12 @@ sub _print
 sub report
 {
     my $self = shift;
+    my $i=0;
+    # Ensure that there is no undefined arg: replace by '' if any
+    while ( $i < scalar(@_)) {
+      $_[$i] = '' if !defined($_[$i]);
```

Comment:
I tend to disagree... See https://github.com/quattor/maven-tools/issues/74#issuecomment-180246320.